src/components/add-to-a-list/template.test.js

```diff
@@ -171,4 +171,19 @@ describe('Add to a list', () => {
       expect($form.attr('action')).toBe('#addItem')
     })
   })
+
+  describe('Empty, null and missing itemLists', () => {
+    function overrideItemList (itemList) {
+      const params = {...examples.default}
+      params.itemList = itemList
+      return params
+    }
+
+    it('should have the same output for all versions of no items being provided', () => {
+      const outputs = [undefined, null, []].map(itemList => render('add-to-a-list', overrideItemList(itemList)).html())
+
+      expect(outputs[0]).toEqual(outputs[1])
```

Comment:
Are there any other places that could potentially have this check? Not expecting it for this PR but just wondering for future improvements.

Reply:
I was surprised to find that this is the only `type: array` that's documented (in the YAML) and I went through all the component sources to look for more - I can't find any.

It's a check worth adding any time a component accepts an array.

Reply:
Ok, thanks for looking through! I'll keep an eye out for any future components.